### PR TITLE
organisations/forms: move brackets to right place to make string tran…

### PR DESCRIPTION
**Pull request by Rineee**
…slatable

fixes https://github.com/liqd/adhocracy-plus/issues/2221
closes https://github.com/liqd/adhocracy-plus/pull/2225

### CODE REVIEW

File: apps/organisations/forms.py [hunk 1]
```diff
@@ -314,10 +314,8 @@ def __init__(self, project=None, format=None, *args, **kwargs):
             required=True,
             help_text=_(
                 "This title will be displayed as a header. "
-                "It should be max. {} characters long.".format(
-                    self.sizes["title_max_length"]
-                )
-            ),
+                "It should be max. {} characters long."
```
Comment (fuzzylogic2000) [inline apps/organisations/forms.py:317]: Ah!

File: locale-source/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -893,24 +893,24 @@ msgid "seconds"
...
 #: adhocracy4/modules/models.py:227
-msgctxt "abbreviation for hour(s)"
+msgctxt "abbreviation \"hour(s)\""
 msgid "H"
-msgstr "S"
+msgstr ""
 
 #: adhocracy4/modules/models.py:229
-msgctxt "abbreviation for minute(s)"
+msgctxt "abbreviation \"minute(s)\""
```
Comment (fuzzylogic2000) [inline locale-source/locale/de/LC_MESSAGES/django.po:906]: This is weird, when I do an install and then do the "makemesages", these explanations are not changed and the translations not lost.

**Review by fuzzylogic2000 — APPROVED** [review]
Supercool!
I will redo the makemessages and then also push to transifex!

File: locale-source/locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (Rineee) [reply locale-source/locale/de/LC_MESSAGES/django.po]: Oh, the translations were also lost? Thats not good..

**Pull request merged**